src/components/board/Board.jsx

```diff
@@ -1,3 +1,34 @@
-const Board = () => {};
+import Image from "next/image";
+const Board = ({ name, role, yearMajor, funFact, picture }) => {
+  return (
+    <div className="flex justify-center w-64 h-96 bg-biscuits-yellow shadow-[20px_-15px_0px_rgb(190,156,106)]">
+      <div>
```

Comment:
remove the divider tag. this will make the text messed up, but that's because its flex-row by default. you can add change it to flex-col in the parent instead.

Reply:
which divider tag should be removed? Also do you add flex col on the furthest outside div?

Reply:
diver tags in line 5 and line 29 (closing tag)

second question, yes.